packages/app/app/containers/ArtistViewContainer/hooks.ts

```diff
@@ -47,13 +48,41 @@ export const useArtistViewProps = () => {
     dispatch(FavoritesActions.removeFavoriteArtist(artist));
   }, [artist, dispatch]);
 
+  const handleArtistInfoClick = () => {
```

Comment:
I think it would be better if getting this link was implemented in the metadata provider, this way we could get it depending on the currently selected provider.

Could you also please add a test for this functionality so it's covered?

Reply:
I'll work on this tonight. Sorry for the delay, the holiday week was more hectic than I imagined. Could you point me towards the right place for the metadata provider? I want to make sure I'm putting stuff in the right place.

Reply:
For example here: https://github.com/nukeop/nuclear/blob/master/packages/core/src/plugins/meta/bandcamp.ts

You should add a method for getting an artist link to the MetaProvider abstract class (https://github.com/nukeop/nuclear/blob/master/packages/core/src/plugins/metaProvider.ts), and implement it in the providers that extend it, or just return null otherwise.

Reply:
![image](https://user-images.githubusercontent.com/47801590/144167824-da36aed8-37e1-485f-93b5-da0928638661.png)
I noticed that the metadata plugins are already returning a resourceUrl as part of the SearchResultsArtist enum, which is used to render the artist search results (if I'm understanding the code correctly). Do you think it would make more sense to pull the URL from there instead of writing a separate function to grab the URL once more?

Reply:
I've added a test for the button similar to the tests for clicking on similar artists or artist tags